### DISCUSSION TITLE
Is there a way to use GUI apps on lima?

### DISCUSSION THREAD

**Discussion by Keffori** [lima-vm/lima#2833] [Q&A] [answered]
I installed AyuGram in lima, tried running it, but got an error: "Error: no DISPLAY environment variable specified". I heard that there's something called XQuartz that can help, but I don't know how to install and use it. I'm a linux noob.

**Comment by AkihiroSuda** [comment, marked as answer]
`ssh -X` (or `ssh -Y`) may work?

```
ssh -X -F $HOME/.lima/default/ssh.config lima-default
```

**Reply by AkihiroSuda** [reply]
You may also use `limactl start --video` to enable the video display

**Reply by Keffori** [reply]
How do I disable --video? It launches with it every time now, when I don't want it

**Reply by Keffori** [reply]
I mean, that there's QEMU window that I don't want to pop up

**Reply by AkihiroSuda** [reply]
`limactl start --video=false`

**Reply by vktt** [reply]
> You may also use `limactl start --video` to enable the video display

OK this works well. It launches a nice window with tty1 text login. But I am now stuck at login. Even though I tried to change the password using `$ sudo passwd`, I cannot log in. What am I missing?

Edit: Adding a new user, and logging in as that user works.
